draft-ietf-httpbis-http2bis.xml

```diff
@@ -1703,7 +1703,7 @@ HEADERS Frame {
           <name>PRIORITY Frame Format</name>
           <artwork type="inline"><![CDATA[
 PRIORITY Frame {
-  Length (24),
+  Length (24) = 0x5,
```

Comment:
There were already a few places where we listed the hardcoded stream identifier in the presentation language, so I figured it was fair game for including statically known values in the presentation language, even when it's part of the generic frame header.  This is also a separate commit for ease of reverting, if needed.